src/components/Models/models.util.js

```diff
@@ -186,27 +194,22 @@ export const checkForSelectedModel = (
   modelName,
   setSelectedModel
 ) => {
-  const [searchItem] = models.filter(item => item.key === modelName)
+  let artifacts = []
+
+  if (models.selectedRowData.content[modelName]) {
+    artifacts = models.selectedRowData.content[modelName]
+  } else {
+    artifacts = models.allData
+  }
```

Comment:
Equivalent to:
```suggestion
    artifacts = models.selectedRowData.content[modelName] || models.allData
```
Or:
```suggestion
    artifacts = models.selectedRowData.content[modelName] ?? models.allData
```